File: DataLayer/Entities/Detection.cs
using System;
using System.Collections.Generic;
using DataLayer.Entities.Common;

namespace DataLayer.Entities
{
    public class Detection : EntityBase
    {
        public string Name { get; set; }
        public int StatusId { get; set; }
        public Status Status { get; set; }
        public DateTime? CompletionTime { get; set; }
        public ImageAttachment Image { get; set; }
        public IEnumerable<DetectionResult> Results { get; set; }
    }
}

File: dataLayer/entities/detection.py
from sqlalchemy import Column, String, Integer, Date
from sqlalchemy.orm import relationship

from dataLayer.database_connection import Base


class Detection(Base):
    __tablename__ = 'Detection'

    id = Column('Id', Integer, primary_key=True)
    dnnFaces = Column('DnnFaces', Integer)
    haarFaces = Column('HaarFaces', Integer)
    name = Column('Name', String)
    statusId = Column('StatusId', Integer)
    image = relationship("ImageAttachment")
    completionTime = Column('CompletionTime', Date)
